script/Adapters.s.sol

```diff
@@ -57,7 +79,14 @@ contract Adapters is Script, JsonRegistry {
         if (vm.parseJsonBool(config, "$.adapters.axelar.deploy")) {
             address gateway = vm.parseJsonAddress(config, "$.adapters.axelar.gateway");
             address gasService = vm.parseJsonAddress(config, "$.adapters.axelar.gasService");
-            AxelarAdapter axelarAdapter = new AxelarAdapter(multiAdapter, gateway, gasService, msg.sender);
+            AxelarAdapter axelarAdapter = AxelarAdapter(
+                create3(
+                    generateSalt("axelarAdapter", version),
+                    abi.encodePacked(
+                        type(AxelarAdapter).creationCode, abi.encode(multiAdapter, gateway, gasService, msg.sender)
+                    )
+                )
+            );
             adapters.push(axelarAdapter);
```

Comment:
CR, IMO we need to add this line for the wormhole adapter too.

Reply:
Fixed in #508